client/blocks/reader-subscription-list-item/email-settings.jsx

```diff
@@ -37,11 +54,37 @@ class ReaderEmailSubscriptionSettingsPopout extends Component {
 	}
 
 	setSelected = text => () => {
+		const { siteId } = this.props;
 		this.setState( { selected: text } );
+		this.props.updateNewPostEmailSubscription( siteId, text );
+	}
+
+	toggleNewPostEmail = () => {
+		const { notifyOnNewPosts, siteId } = this.props;
+
```

Comment:
this could also be 
```
const action = notifyOnNewPosts ? this.props.unsubscribeToNewPostEmail : this.props.subscribeToNewPostEmail;
action( siteId );
```